resources/qml/Settings/SettingTextField.qml

```diff
@@ -106,7 +106,16 @@ SettingItem
             {
                 target: input
                 property: "text"
-                value: propertyProvider.properties.value
+                value:  {
+                    if ((propertyProvider.properties.resolve != "None") && (stackLevel != 0)) {
```

Comment:
So only if the user-setting is not set we return the resolve? So if the user sets the setting and doesn't save it, the resolve is not used. When the user sets the setting and makes a quality change out of it, the resolve is used?